serverless/pages/serverless-differences.asciidoc

```diff
@@ -26,6 +26,32 @@ This guide helps you understand what's different, what's available, and how to w
 
 This fully managed approach means many traditional {es} infrastructure APIs and settings are not available to end users, as detailed in the following sections.
 
+[discrete]
+[[elasticsearch-differences-serverless-index-size]]
+== Index size guidelines
+
+Please note the following recommendations for sizing individual indices for optimal performance on {es-serverless}:
+
+|===
+|Use case |Maximum index size |Project configuration
+
+|Vector search
+|150GB
+|Vector optimized
+
+|General search (non data-stream)
+|300GB
+|General purpose
+
+|Other uses (non data-stream)
+|600GB
+|General purpose
+|===
+
+For large datasets that exceed the recommended maximum size for a single index, consider splitting your data across smaller indices and using an alias to search them collectively.
+
+NOTE: These recommendations do not apply to indices using Better binary quantization (BBQ). Refer to {ref}/dense-vector.html#dense-vector-quantization[vector quantization] in the core {es} docs for more information.
```

Comment:
prefer only capitalizing product names. suggest taking this out of a note because it's key info

```suggestion
These recommendations do not apply to indices using better binary quantization (BBQ). Refer to {ref}/dense-vector.html#dense-vector-quantization[vector quantization] in the core {es} docs for more information.
```